README.md

```diff
@@ -55,6 +55,10 @@ const pool = new Pool({
   // if a resource cannot be acquired
   createTimeoutMillis: 30000,
 
+  // destroy operations are awaited for at most this many milliseconds
+  // new resources will be created after this timeout
+  destroyTimeoutMillis: 200,
```

Comment:
This sounds way too small timeout, better to wait few seconds to be sure. I wouldn't think that waiting for example 5 seconds in case of weird problems here would be a problem. Better to wait a bit longer than timeout too soon, before destroy is ready.

Reply:
should be changed to be 5000 here too to be consistent